### ISSUE TITLE
The Mule is detected as a contact by the Liberia.

### ISSUE THREAD

**Issue by MistakeNot4892** [NebulaSS13/Nebula#2710]
#### Description of issue
On load, the Liberia goes through the rigmarole of detecting and logging its own shuttle as an overmap contact.

#### Difference between expected and actual behavior
It should probably be aware of its own shuttle.

#### Steps to reproduce
Load in on the Liberia bridge.

#### Specific information for locating
Fairly sure this is due to the Mule being a fully-fledged overmap entity rather than a docked shuttle. Might be worth converting it into something like the exploration shuttle on the tradeship. Otherwise we'll need to add some way for overmap objects to know about each other ahead of time.

#### Length of time in which bug has been known to occur
No clue.

#### Client version, Server revision & Game ID
Client Version: 514
Server Revision: 9be20531a3eab4e7f1598ac79d1e03f4b5cf298e - awaysites - 2022-10-03
Game ID: ckd-cfQ5
Current map: Tradeship Ocelot Alpha

Basically `staging`.

#### Issue bingo
- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)
